app/src/main/java/org/schabi/newpipe/views/LongPressLinkMovementMethod.java

```diff
@@ -0,0 +1,77 @@
+package org.schabi.newpipe.views;
```

Comment:
And also this I would say, so that we have everything related to sharing, spans, copying to clipboard, ... in a single folder

Reply:
Ok, makes sense